backend/config/redis.js

```diff
@@ -0,0 +1,20 @@
+import dotenv from 'dotenv';
```

Comment:
configs are not used for clients, better to use service for redis client.
To get environment-vars, you can use config.